src/salary-app/grossToNet.test.js

```diff
@@ -1,47 +1,51 @@
-const assert = require('assert');
-const test = require('mocha').it;
-const sinon = require('sinon');
-const proxyquire = require("proxyquire");
+import assert from 'assert';
+import {it as test} from 'mocha'
+import sinon from 'sinon';
+import proxyquire from 'proxyquire';
 
-describe("Calculate net salary", function () {
+describe("Calculate net salary", () => {
     let insurancesCalcStub;
     let taxCalcStub;
-    let net;
+    let grossToNet;
 
-    beforeEach(function () {
-        insurancesCalcStub = sinon.stub();
+    beforeEach(() => {
+        insurancesCalcStub = sinon.stub() ;
         taxCalcStub = sinon.stub()
 
-        net = proxyquire("../src/net.js", {
-            "./insurancesCalculator.js": insurancesCalcStub,
-            "./taxCalculator.js": taxCalcStub,
-        });
+        grossToNet = proxyquire("./grossToNet", {
+            './common/calcInsuranceDetails': {
+                default: insurancesCalcStub
+            },
+            "./common/calcTaxes": {
+                default: taxCalcStub
+            } ,
+        }).default;
     });
 
-    afterEach(function () {
+    afterEach(() => {
         sinon.restore();
     })
 
-    describe('Check Dependent deduction', function () {
+    describe('Check Dependent deduction', () => {
         //Personal Income Tax (PIT)
         const mockedInsurance = { total: 0, insurances: [] };
         const mockedTaxes = { totalTax: 0, rates: [] };
-        beforeEach(function () {
+        beforeEach(() => {
             insurancesCalcStub.returns(Promise.resolve(mockedInsurance));
             // insurancesCalcStub.resolves(mockedInsurance);
 
             taxCalcStub.returns(
-                new Promise(function (resolve, reject) {
+                new Promise((resolve, reject) => {
```

Comment:
There is a shorter way by using static method of Promise class: `Promise.resolve(mockedTaxes)`

Reply:
There are 3 ways:
```JavaScript
 insurancesCalcStub.returns(Promise.resolve(mockedInsurance));
            // insurancesCalcStub.resolves(mockedInsurance);

            taxCalcStub.returns(
                new Promise((resolve, reject) => {
                    resolve(mockedTaxes);
                })
            )
```